client/src/components/NationwideMap.vue

```diff
@@ -106,23 +107,37 @@ export default defineComponent({
     height: { type: String, default: '80vh' },
   },
   methods: {
+    /**
+     * Convert lat,long strings to MapBox LngLatLike
+     */
+    getLngLatLikeFromLatLong(lat: string, long: string): LngLatLike {
+      return {
+        lon: parseFloat(long),
+        lat: parseFloat(lat),
+      };
+    },
     zoomToLongLat() {
-      if (this.geoState?.geoids?.pwsId?.bounding_box) {
-        const {
-          minLat,
-          minLon,
-          maxLat,
-          maxLon,
-        } = this.geoState?.geoids?.pwsId?.bounding_box;
-        this.map?.fitBounds([
-          [minLat, minLon],
-          [maxLat, maxLon],
-        ]);
-      } else if (this.geoState?.geoids?.lat != null && this.geoState?.geoids.long != null) {
-        const lonLat: LngLatLike = {
-          lon: parseInt(this.geoState?.geoids?.long),
-          lat: parseInt(this.geoState?.geoids?.lat),
-        };
+      const addressBoundingBox = this.geoState?.geoids?.address?.boundingBox;
+      const waterSystemBoundingBox = this.geoState?.geoids?.address?.boundingBox;
+
+      const lat = this.geoState?.geoids?.lat;
+      const long = this.geoState?.geoids?.long;
+
+      // If there's an address to zoom to, choose that.
+      if (addressBoundingBox != null && lat != null && long != null) {
```

Comment:
I don't yet want to use the bounding box here bc it zooms way too tight (all you see if the parcel). However, I think this will be useful to have

Reply:
Makes sense!